lib/src/Uncertainty/Model/DistributionImplementation.cxx

```diff
@@ -81,10 +81,16 @@
 BEGIN_NAMESPACE_OPENTURNS
 
 CLASSNAMEINIT(DistributionImplementation)
+CLASSNAMEINIT(DistributionImplementation::PDFWrapper)
+CLASSNAMEINIT(DistributionImplementation::LogPDFWrapper)
+CLASSNAMEINIT(DistributionImplementation::CDFWrapper)
 
 typedef Collection<Distribution>                                      DistributionCollection;
 
 static const Factory<DistributionImplementation> Factory_DistributionImplementation;
+static const Factory<DistributionImplementation::PDFWrapper> Factory_PDFWrapper;
```

Comment:
maybe Factory_PDFWrapper -> Factory_DistributionImplementation_PDFWrapper to avoid potential conflicts in the future

Reply:
```suggestion
static const Factory<DistributionImplementation::PDFWrapper> Factory_DistributionImplementation_PDFWrapper;
```